### PR TITLE
feat: Release WebUI templates with notes

### PR DESCRIPTION
**Pull request by gt2345**
## Ticket

[MD-407]

## Description

Default manage templates feature flag to true.
[Link](https://hpe.sharepoint.com/:w:/r/teams/detai/Shared%20Documents/Engineering/Web/ERDs/Web%20implantation%20for%20Templates.docx?d=w21a4382ed90742f59d597425c0899c22&csf=1&web=1&e=trAYN6) to ERD

## Test Plan

CRUD templates should work as expected

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[MD-407]: https://hpe-aiatscale.atlassian.net/browse/MD-407?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/release-notes/webui-manage-templates.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**Improvements**
+
+-  Web UI: Allow user to manage templates through Web UI
```
Comment (tara-det-ai) [inline docs/release-notes/webui-manage-templates.rst:5]: :orphan:

**Improvements**

-  WebUI: Allow users to create and manage configuration templates through the WebUI.

Comment (tara-det-ai) [reply docs/release-notes/webui-manage-templates.rst:5]: @gt2345 i'll also create a docs PR for this to describe the steps

Comment (gt2345) [reply docs/release-notes/webui-manage-templates.rst]: That would be great, could you please stamp this PR so that it's ready for the release tomorrow

**Review by tara-det-ai — APPROVED** [review]
LGTM

**Pull request merged**